merlin-server/sql/merlin/functions/public/begin_merge.sql

```diff
@@ -266,18 +266,18 @@ begin
   union
   -- 'modify' against a 'modify' must be checked for equality first.
   select _merge_request_id, activity_id, name, tags,  source_scheduling_goal_id, created_at,
-         start_offset, type, arguments, metadata, anchor_id, anchored_to_start, 'none'
+         last_modified_by, start_offset, type, arguments, metadata, anchor_id, anchored_to_start, 'none'
   from (
     select activity_id, name, metadata.tag_ids_activity_directive(dd.activity_id, psa.snapshot_id) as tags,
-           source_scheduling_goal_id, created_at, start_offset, type, arguments, metadata, anchor_id, anchored_to_start
+           source_scheduling_goal_id, last_modified_by, created_at, start_offset, type, arguments, metadata, anchor_id, anchored_to_start
       from plan_snapshot_activities psa
       join diff_diff dd
         on dd.activity_id = psa.id
       where psa.snapshot_id = snapshot_id_supplying
         and (dd.change_type_receiving = 'modify' and dd.change_type_supplying = 'modify')
     intersect
     select activity_id, name, metadata.tag_ids_activity_directive(dd.activity_id, ad.plan_id) as tags,
-           source_scheduling_goal_id, created_at, start_offset, type, arguments, metadata, anchor_id, anchored_to_start
+           source_scheduling_goal_id, last_modified_by, created_at, start_offset, type, arguments, metadata, anchor_id, anchored_to_start
```

Comment:
I'm a bit confused on how the merge equality is actually being calculated here. I originally added `last_modified_by` here since we needed the same number of columns for unions / intersections. Where should I update the column otherwise?

Reply:
Change has been made. We decided to add `last_modified_by` into the MSA by grabbing it from the plan snapshot table, so we apply changes to this column during merge commit, but it doesn't count as a "real change" in the diff.

Reply:
> Change has been made. We decided to add `last_modified_by` into the MSA by grabbing it from the plan snapshot table

We want to grab `last_modified_by` from the same place we're loading the activity in from, _not_ exclusively from the plan snapshots table. (The code is doing this correctly, just correcting the mental model for if we need to look at this discussion again in a few months)